cmake/ports/sdl2/CONTROL

```diff
@@ -1,6 +1,5 @@
 Source: sdl2
 Version: 2.0.10-2
-Homepage: https://github.com/SDL-Mirror/SDL
```

Comment:
removed the homepage?

Reply:
Ah I see, in the main branch too,    wouldn't you like to keep those as a reference?

Reply:
I already removed it with fd05144.